lib/meilisearch/rails/templates/initializer.rb

```diff
@@ -0,0 +1,4 @@
+MeiliSearch::Rails.configuration = {
+  meilisearch_url: ENV.fetch("MEILISEARCH_HOST", "YourMeilisearchUrl"), # example: http://localhost:7700
```

Comment:
```suggestion
  meilisearch_url: ENV.fetch("MEILISEARCH_HOST", "http://localhost:7700"),
```

Reply:
Thanks for the feedback!

Nice catch! I'll add it right away.